bbs/lib/mainfunc.php

```diff
@@ -1,11 +1,11 @@
 <?php
+require_once "../../config.php";
 function mainfunc($posts,$debug=false){
 	$ip=$_SERVER["REMOTE_ADDR"];
 	@$token=$_COOKIE['token'];
-	$url="http://127.0.0.1/api/jiekouapi.php?ip=$ip&token=$token";
+	$url=CAPU_JIEKOUAPI."?ip=$ip&token=$token";
```

Comment:
We need to set a rule for using `A . B` or `A.B`. The difference is whether we have space before and after the period. But there is another rule we need to follow: we should only use one style all over the code base.